crates/simulations/src/cozy/statistics/mod.rs

```diff
@@ -0,0 +1,6 @@
+#![allow(dead_code)]
```

Comment:
No need to review stuff in the `statistics` crate. They are just implementations of distributions that were not directly available in a rust crate that we need for the cozy risk model and simulating binary triggers.